### DISCUSSION TITLE
Hey Guys. Virus

### DISCUSSION THREAD

**Discussion by Grastice** [community/community#32827] [Code Security] [answered]
Hey i am sorry for this. I don't know where to talk about this but i think there is someone in this community bad and evil.

He tries to install some cheap malware or some kind of virus in to your computer but thanks to antiviruses that installment prevented.
The download adress points here and so i am too.I will drop the link here guys please DO NOT CLICK ON THIS AND IF ITS POSSIBLE REMOVE IT OR TELL THE ADMINS TO DO SO.

OKAY HERE IS THE BAD LINK 
[Removed due to safety concerns]

**Comment by Nukesor** [comment, marked as answer]
Hey :)

Thanks for the report! Sadly this isn't the right place to report stuff like this.

I went ahead, investigated the repository + the linked file and filed a proper report to Github. I guess this should do the job, but feel free to add another report if you like :).

Here's a non-download view of the repository:
https://github.com/plainsoftupdt/Download-Setup-58

In case somebody else stumbles upon a similar issue, users can be reported via a report button directly below their profile:
![image](https://user-images.githubusercontent.com/3322822/189777215-a739f81e-373a-4293-aefc-0ba74ada2e70.png)

- Select `Report abuse` in the dialog that pops up
- Click on `I want to report harmful code, such as malware, phishing, or cryptocurrency abuse.`
- Select `I want to report malware.`
- Write your text in the textbox below and press `Send request`

**Reply by Nukesor** [reply]
FYI, this is my report:
```
I'm going to file another report for good measure, as I didn't get a confirmation email the first time around (and I don't see this in my support tickets).

A user reported an account that's being used to ship a malware blob.
The original report can be found over here: https://github.com/orgs/community/discussions/32827

The account owners try to obfuscate their activity, by regularily renaming their repository (by incrementing the number :D):
https://github.com/plainsoftupdt/Download-Setup-58

The repository contains a single zip file, which decompresses to a ~700MB obfuscated binary blob.
The blob triggers Window's Antivirus mechanisms and also looks fishy as hell.
```

**Reply by Grastice** [reply]
Thank you very much
